themes/digital.gov/layouts/services/card-service.html

```diff
@@ -15,28 +15,30 @@
   {{/* Logos
     Get the "source" in the file and looks up the icon that is used there.
   */}}
+
+  {{ .Scratch.Set "logo_src" "" }}
   {{- $source := $.Site.GetPage "page" (print "source_" .Params.source ) -}}
+
+  {{/* Find the logo source. */}}
+  {{/* @TODO: Simplify even further. */}}
   {{- if $source -}}
-    {{- $src := (printf "/logos/%s%s" $source.Params.logo "-logo.png") -}}
-    {{- template "logo" dict "src" $src "alt" $source.Params.name "href" $href -}}
+    {{ .Scratch.Set "logo_src" (printf "/logos/%s%s" $source.Params.logo "-logo.png") }}
   {{- else if .Params.logo -}}
-
-    {{- $src := (printf "/logos/%s%s" .Params.logo "-logo.png") -}}
-    {{- template "logo" dict "src" $src "alt" .Params.name "href" $href -}}
-
+    {{ .Scratch.Set "logo_src" (printf "/logos/%s%s" .Params.logo "-logo.png") }}
   {{- else -}}
-
-    {{- template "logo" dict "src" "/logos/digit-gold.png" "alt" .Params.source "href" $href -}}
-
+    {{ .Scratch.Set "logo_src" "/logos/digit-gold.png" }}
   {{- end -}}
 
+  {{/* Call logo template once with new found source. */}}
+  {{- template "logo" dict "src" (.Scratch.Get "logo_src") "href" $href -}}
+
 
   <div class="copy">
-    <h3>
+    <h2 class="card__title">
```

Comment:
Class for styling instead of element to prevent markup changes from breaking styles.